packages/client/src/components/LeaderboardPage/types.ts

```diff
@@ -0,0 +1,7 @@
+export type leaderDataType = {
```

Comment:
Название типа обычно делают с большой буквы. И предлагаю не в конце приписывать Type, а в начале букву T - так короче будет))

```suggestion
export type TLeaderData = {
```